File: app/src/main/java/calendario/kevshupp/diariokevinali/AppNavigation.java
package calendario.kevshupp.diariokevinali;

public interface AppNavigation {
    void pickImage(int requestCode);
    void logout();
    void showAddEventDialog(String date, CalendarEvent event);
    String getCurrentTheme();
    void applyTheme(String theme);
    void showUpdateDialog(String url);
}

File: app/src/main/java/calendario/kevshupp/diariokevinali/AppNavigation.kt
package calendario.kevshupp.diariokevinali

interface AppNavigation {
    fun pickImage(requestCode: Int)
    fun logout()
    fun showAddEventDialog(date: String, event: CalendarEvent?)
    fun getCurrentTheme(): String
    fun applyTheme(theme: String)
    fun showUpdateDialog(url: String)
}
